server/test/mockTest.spec.js

```diff
@@ -0,0 +1,208 @@
+const chai = require("chai");
+const chaiHttp = require("chai-http");
```

Comment:
Change the name of the file to represent the type of test you are running (as opposed to mockTest)